lib/ioc-rc

```diff
@@ -534,14 +538,15 @@ __stop_jail () {
     local _fulluuid _dataset _name _jail_path _tag _exec_prestop _exec_stop \
           _exec_poststop _vnet _state _ip4 _jail_type _jail_zfs \
           _jail_zfs_datasets _zfs_dataset _zfs_dataset_subsets \
-          _zfs_dataset_subset _nfulluuid
+          _zfs_dataset_subset _nfulluuid _no_unmount
 
     if [ "$1" = "ALL" ] ; then
         __stop_jail_all
         exit $?
     fi
 
     _name=$1
+    _no_unmount=$2
```

Comment:
but not around $1? can we stop this ridiculousness and keep going forward, please?